FLEDGE.md

```diff
@@ -109,7 +109,7 @@ The browser will remain in an interest group for only a limited amount of time.
 
 The `user_bidding_signals` is for storage of additional metadata that the owner can use during on-device bidding, and the `trusted_bidding_signals_*` attributes provide another mechanism for making real-time data available for use at bidding time.
 
-The `daily_update_url` provides a mechanism for the group's owner to periodically update the attributes of the interest group: any new values returned in this way overwrite the values previously stored (except that the `name` and `owner` cannot be changed).  However, the browser will only allow daily updates for interest groups with a sufficiently large number of people, e.g. at least 100 browsers with the same update url.  (Without this sort of limit, a single-person interest group could be used to observe that person's coarse-grained IP-Geo location over time.)
+The `daily_update_url` provides a mechanism for the group's owner to periodically update the attributes of the interest group: any new values returned in this way overwrite the values previously stored (except that the `name` and `owner` cannot be changed).  However, the browser will only allow daily updates when a sufficiently large number of people have the same `daily_update_url` , e.g. at least 100 browsers with the same update url. This will not include any metadata, so data such as the interest group `name` should be included within the url, so long as url exceeds the minimum size constraint.  (Without this sort of limit, a single-person interest group could be used to observe that person's coarse-grained IP-Geo location over time.)
```

Comment:
How about "exceeds the minimum count threshold" instead of "exceeds the minimum size constraint".

Reply:
updated 🙂